defaults/main.yml

```diff
@@ -114,3 +114,13 @@ kops_default_kubeControllerManager:
     - TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
 kops_default_kubeScheduler:
   enableProfiling: false
+
+###
+### Verify dockerconfig via get ratelimited
+### https://docs.docker.com/docker-hub/download-rate-limit/#how-can-i-check-my-current-rate
+###
+kops_default_dockerconfig:
```

Comment:
This defines a default, but when is this default ever used?

Reply:
we still need the url and timeout value as default value